src/expressions.rst

```diff
@@ -3431,6 +3431,14 @@ A :t:`closure body` denotes a new :t:`control flow boundary`.
 :dp:`fls_fg8lx0yyt6oq`
 A :t:`closure body` is subject to :t:`capturing`.
 
+:dp:`fls_c3rzwUxjmBMY`
+A :t:`closure parameter` is a :t:`construct` that matches an input :t:`value`
+at the site of a :t:`call expression` or a :t:`method call expression` to
+a pattern.
```

Comment:
This might need a glossary entry. I don't see a definition of `closure parameter` in Expressions.